Content.Server/Body/Systems/InternalsSystem.cs

```diff
@@ -38,6 +40,38 @@ public override void Initialize()
         SubscribeLocalEvent<InternalsComponent, ComponentShutdown>(OnInternalsShutdown);
         SubscribeLocalEvent<InternalsComponent, GetVerbsEvent<InteractionVerb>>(OnGetInteractionVerbs);
         SubscribeLocalEvent<InternalsComponent, InternalsDoAfterEvent>(OnDoAfter);
+        SubscribeLocalEvent<BreathToolComponent, ConnectedBreathToolEvent>(OnBreathMaskConnected);
+    }
+
+    public override void Update(float frameTime)
+    {
+        base.Update(frameTime);
+
+        var query = EntityQueryEnumerator<InternalsDelayedActivationComponent>();
+        while (query.MoveNext(out var uid, out var delayComp))
+        {
+            if (_timing.CurTime < delayComp.Time)
+                return;
+
+            if (TryComp<InternalsComponent>(uid, out var comp))
+                ToggleInternals(delayComp.Entity , delayComp.Entity , false, comp);
```

Comment:
You should probably check that the internals are not already active before toggling them on. That shouldn't be the case but it may prevent future bugs/unexpected behavior.

Reply:
I figured such a check wouldn't be needed since internals being connected means it has just been reset, but you are right. It's better to make things more future-safe